style.css

```diff
@@ -1,2 +1,86 @@
+.credit-card  {
+    width: 500px;
+    height: 325px;
+    background-color: royalblue;
+    border-radius: 20px 20px 20px 20px;
+    display: flex;
+    flex-direction: column;
+    font-family: "Geologica", sans-serif;
+    font-weight: 300;
+    font-optical-sizing: auto;
+}
 
+.credit-card-front {
+    display: flex;
+    justify-content: flex-end;
+}
 
+.credit-card-front .information-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 0 0 10px 39px;
+    font-size: 20px;
+    width: 20%;
+    height: 20%
+}
+
+.information-wrapper > p:nth-child(1) {
+    font-weight: lighter;
+    text-transform: uppercase;
+}
+
+.information-wrapper > p:nth-child(2) {
+    margin-left: 10px;
+    font-size: 30px;
+}
+
+.credit-card-front .number {
+    display: flex;
+    justify-content: center;
+    font-size: 40px;
+}
+
+.credit-card-front .name {
+    font-size: 35px;
+    margin: 0 0 25px 38px;
+}
+
+.credit-card-back {
+    margin-top: 20px;
+}
+
+.magnetic-strip {
+    height: 18%;
+    background-color: black;
+    margin-top: 35px;
+}
+
+.credit-card-back .information-wrapper {
+    width: 55%;
+    height: 17%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    background-color: lightgray;
+    margin: 30px 0 0 15px;
+}
+
+.credit-card-back .name {
+    width: 100%;
+    font-family: "Sacramento", cursive;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 20px;
+    margin: 15px 0 0 25px;
+}
+
+.credit-card-back .number.number {
+    width: 30%;
+    height: 100%;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0;
+    font-size: 20px;
+    font-weight: lighter;
```

Comment:
If you take a closer look, there is one CSS property missing that you should apply. This text should be in italic.

![Screenshot from 2024-03-08 12-08-25](https://github.com/mikolajkus/Credit-card/assets/15181215/c716c43a-cd03-4250-89ee-8e3770aff1aa)

Reply:
I didn't notice it.